test/lender/Lender.sol

```diff
@@ -9,27 +9,27 @@ import './Safe.sol';
 import './Cast.sol';
 import './Element.sol';
 
+
 contract Lender {
   address public admin;
   address public illuminate; // TODO authorized setter for this
 
-  // TODO the nature of these addresses?
   address public swivelAddr; // addresses of the 3rd party protocol contracts
-  address public pendleRouter;
-  address public tempusRouter;
+  address public pendleAddr;
+  address public tempusAddr;
   address public senseToken;
 
   event Lend(uint8 principal, address indexed underlying, uint256 indexed maturity, uint256 returned);
   event Mint(uint8 principal, address indexed underlying, uint256 indexed maturity, uint256 amount);
 
   /// @param i the deployed Illuminate contract
-  constructor(address i, address s, address su, address t, address sa) {
+  constructor(address i, address s, address pr, address t, address st) {
     admin = msg.sender;
     illuminate = i; // TODO add an authorized setter for this?
     swivelAddr = s;
-    pendleRouter = su;
-    tempusRouter = t;
-    senseToken = sa;
+    pendleAddr = pr;
+    tempusAddr = t;
+    senseToken = st;
```

Comment:
this should be `senseAddr`. the `...Token` naming schemes are used for `IFooToken` interfaces

Reply:
In this case, `senseToken` is used the same way other tokens are used. However, I think we are not using `senseToken` correctly. I am going to remove this reference and create a ticket to double check this.